tests/unit/utils/text-test.ts

```diff
@@ -0,0 +1,10 @@
+import { slugify} from 'open-event-frontend/utils/text';
```

Comment:
Codacy found an issue: [A space is required before '}'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5730106)